include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
I would prefer #including TestHarness.h as indicated above. That way, you don't need to patch `StandardCLibrary.h` in this unusual way.

Including `StandardCLibrary.h` directly anywhere is non-standard and, in my opinion, shouldn't be done. So, even if you wish to keep this patch (for added safety) I vote to include `Testharness.h` as indicated above.